notebooks/scikit-learn/Training and Prediction with scikit-learn.ipynb

```diff
@@ -57,7 +57,6 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import datetime\n",
     "import pandas as pd\n",
     "\n",
     "from google.cloud import storage\n",
```

Comment:
If the goal is for these notebooks to run completely independent of GCP, we should remove any mention or usage of GCP products.